### DISCUSSION TITLE
[PSD5] Exercise 2.2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#139] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210014525-bdd366ed-a6cd-43df-969f-d62f8eca49ee.png)

**Comment by Elia-Belli** [comment]
**Forma Canonica**: non minimizzata

(a) $\sum{(1,2,3)} = \bar A B + A \bar B + A B$
(b) $\sum{(1,2,3,4,6)} = \bar A \bar B C+ \bar A B \bar C+ \bar A B C+ A \bar B \bar C+ A B \bar C$
(c) $\sum{(1,6,7)} = \bar A \bar B C+ A B \bar C+ A B C$
(d) $\sum{(0,2,3,6,7,8,10)} = \bar A \bar B \bar C \bar D+ \bar A \bar B C \bar D+ \bar A \bar B C D+ \bar A B C \bar D+ \bar A B C D + A \bar B \bar C \bar D+ A \bar B C \bar D$
(e) $\sum{(3,6,7,8,9,10,11)} = \bar A \bar B C D+ \bar A B C \bar D+ \bar A B C D+ A \bar B \bar C \bar D+ A \bar B \bar C D+ A \bar B C \bar D+ A \bar B C D$

**Comment by FeddyLix17** [comment]
tutto corretto

**Comment by niccolo-fato** [comment]
![photo_2024-11-16_20-54-02](https://github.com/user-attachments/assets/68e1d0c8-6ac9-422a-89a8-9223c5f9fb49)
